src/utils/components/Consoles/components/CloudInitCredentials/InlineCodeClipboardCopy.tsx

```diff
@@ -3,21 +3,32 @@ import React, { FC } from 'react';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ClipboardCopy } from '@patternfly/react-core';
 
-import { clipboardCopyFunc } from '../../utils/utils';
+type InlineCodeClipboardCopyProps = {
+  clipboardText: string;
+  hideText?: boolean;
+};
 
-const InlineCodeClipboardCopy: FC<{ clipboardText: string }> = ({ clipboardText }) => {
+const InlineCodeClipboardCopy: FC<InlineCodeClipboardCopyProps> = ({
+  clipboardText,
+  hideText = false,
+}) => {
   const { t } = useKubevirtTranslation();
 
+  const handleCopy = () => {
+    navigator.clipboard.writeText(clipboardText);
```

Comment:
the write may fail due to browser restrictions and/or missing features - to avoid unhandled promise rejection errors we can add a catch handler and log some meaningful message